store/postgres/src/subgraphs.graphql

```diff
@@ -43,6 +43,12 @@ type SubgraphDeployment @entity {
     graftBase: SubgraphDeployment
     graftBlockHash: Bytes
     graftBlockNumber: BigInt
+
+    # Tracking of reorgs so that we can spot that a reorg
+    # happened while a query was being processed
+    reorgCount: Int
+    currentReorgDepth: Int
+    maxReorgDepth: Int
```

Comment:
When I make that `Int!`, I get a bunch of test failures because the storage layer requires that new entities have a value for these fields; but for these fields that's not necessary, since the columns have a `default 0` clause.

Reply:
Oh got it